.storybook/webpack.config.ts

```diff
@@ -1,8 +1,15 @@
 /* eslint-env node */
 /* eslint import/no-nodejs-modules:0 */
-const path = require('path');
-const webpack = require('webpack');
-const appConfig = require('../webpack.config');
+
+import path from 'path';
+
+import webpack from 'webpack';
+
+import appConfigAny from '../webpack.config';
+
+// TODO(epurkhiser): Once we convert our webpack.config over to typescript we
+// can remove this
+const appConfig = appConfigAny as any;
```

Comment:
This will go away once we get webpack config in ts. If I don't any it right now it uses the `as const` version of the raw js file, which has poorly interpreted types.